nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1455,25 +1455,27 @@ The default encryption algorithm utilized is AES/GCM 128/256-bit. 128-bit is use
 
 You can use the following command line options with the `encrypt-config` tool:
 
```

Comment:
The ordering is changed here (to match to ordering of the usage output when running the tool), so it looks like a larger change. There are only two new options:

* `-a`,`--authorizers <arg>`  The authorizers.xml file containing unprotected config values (will be overwritten)
* `-u`,`--outputAuthorizers <arg>` The destination authorizers.xml file containing protected config values (will not modify input authorizers.xml)